generate.py

```diff
@@ -23,11 +23,15 @@ def read_json_files(folder):
     for filename in os.listdir(folder):
         if filename == '.gitkeep':
             continue
-        #print(filename)
+        elif not filename.endswith('.json'):
```

Comment:
given that the previous `if` ends with `continue` this should be also an `if` and not `elif`